mobile/library/swift/EngineBuilder.swift

```diff
@@ -546,6 +557,74 @@ open class EngineBuilder: NSObject {
     return self
   }
 
+  /// Sets the node.id field in the Bootstrap configuration.
+  ///
+  /// - parameter nodeId: The node ID.
+  ///
+  /// - returns: This builder.
+  @discardableResult
+  public func setNodeId(_ nodeId: String) -> Self {
+    self.nodeId = nodeId
+    return self
+  }
+
+  /// Sets the node locality in the Bootstrap configuration.
+  ///
+  /// - parameter region:  The region.
+  /// - parameter zone:    The zone.
+  /// - parameter subZone: The sub-zone.
+  ///
+  /// - returns: This builder.
+  @discardableResult
+  public func setNodeLocality(
+    _ region: String,
+    _ zone: String,
+    _ subZone: String
+    ) -> Self {
+    self.nodeRegion = region
+    self.nodeZone = zone
+    self.nodeSubZone = subZone
+    return self
+  }
```

Comment:
Similarly, we should follow Swift's API guidelines here:

```suggestion
  public func setNodeLocality(
    region: String,
    zone: String,
    subZone: String
  ) -> Self {
    self.nodeRegion = region
    self.nodeZone = zone
    self.nodeSubZone = subZone
    return self
  }
```